model/feature_flags.go

```diff
@@ -72,6 +72,9 @@ type FeatureFlags struct {
 	BoardsDataRetention bool
 
 	NormalizeLdapDNs bool
+
+	// Enable health of server dashboard
+	HealthOfServerDashboard bool
```

Comment:
It is now "WorkspaceOptimizationDashboard" ... could you change that to the new name please? 👍🏻🙇🏼‍♂️